### DISCUSSION TITLE
Transaction pending on my device for months

### DISCUSSION THREAD

**Discussion by Merrily6969** [community/community#129492] [Mobile]
### Select Topic Area

General

### Body

I had a transaction of around $60 in Bitcoin sent to the wallet on my phone and it's still pending for much it says it's a space issue but it might be money too please help me get this transaction to load on your wallet that you provided for me to download on my phone or a different wallet that I could transfer it to you

**Comment by eliperkins** [comment]
Thanks for starting a discussion in the GitHub community. This looks like it's not quite related to the [GitHub Mobile product](https://github.com/mobile), which is what the "Mobile" category here is about. I'll close this discussion out as "off-topic" as such.

Best of luck in getting this resolved.
